modules/core/src/main/java/org/apache/ignite/spi/communication/tcp/TcpCommunicationSpi.java

```diff
@@ -353,38 +249,19 @@
     public static final int DFLT_MSG_QUEUE_LIMIT = GridNioServer.DFLT_SEND_QUEUE_LIMIT;
 
     /**
-     * Default count of selectors for TCP server equals to
-     * {@code "Math.max(4, Runtime.getRuntime().availableProcessors() / 2)"}.
+     * Default count of selectors for TCP server equals to {@code "Math.max(4, Runtime.getRuntime().availableProcessors()
+     * / 2)"}.
```

Comment:
Let's avoid linefeed in the middle of 'Runtime.getRuntime().availableProcessors() / 2'.
or make javadoc a bit shorter "Default count of TCP server selectors is ..."